chapters/sampling-randomization-power.tex

```diff
@@ -392,7 +386,7 @@ \section{Power calculation and randomization inference}
 you can find some possible subsets that have higher-than-average values of some measure;
 similarly, you can find some that have lower-than-average values.
 Your sample or randomization will inevitably fall in one of these categories,
-and we need to assess the likelihood and magnitude of this occurence.\sidenote{
+and we need to assess the likelihood and magnitude of this occurrence.\sidenote{
```

Comment:
Did not expect to find edits this later into a chapter in this lede PR. Just want to make sure that this was intentional.

Reply:
Same as above. The correction is correct anyway!